ui/components/component-library/banner-base/banner-base.types.ts

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import type {
+  PolymorphicComponentPropWithRef,
+  StyleUtilityProps,
+} from '../box';
+
+export interface BannerBaseStyleUtilityProps extends StyleUtilityProps {
+  /**
+   * The title of the BannerBase
+   */
+  title?: string;
+  /**
+   * Additional props to pass to the `Text` component used for the `title` text
+   */
+  titleProps?: any;
+  /**
+   * The description is the content area below BannerBase title
+   */
+  description?: string;
+  /**
+   * Additional props to pass to the `Text` component used for the `description` text
+   */
+  descriptionProps?: any;
+  /**
+   * The children is an alternative to using the description prop for BannerBase content below the title
+   */
+  children?: React.ReactNode;
+  /**
+   * Label for action button (ButtonLink) of the BannerBase below the children
+   */
+  actionButtonLabel?: string;
+  /**
+   * Props for action button (ButtonLink) of the BannerBase below the children
+   */
+  actionButtonProps?: any;
```

Comment:
`actionButtonProps?: ButtonLinkProps<'button'>;` ?

Reply:
Hey @dhruvv173, that makes sense to me thanks!